Patches/GameOptionsMenuPatch.cs

```diff
@@ -795,9 +803,37 @@ private static void CreateCustomQuotaOption(RolesSettingsMenu inst, RoleBase dog
         List<UiElement> quotaMono = inst.QuotaTabSelectables.ToManaged();
         quotaMono.AddRange(roleOptionSetting.ControllerSelectable.ToManaged());
         inst.QuotaTabSelectables = quotaMono.ToIl2Cpp();
+        OnValueChanged(roleOptionSetting, dog);
         yPos -= 0.43f;
     }
 
+    private static void IncreaseChance(OptionBehaviour option, RoleBase role)
+    {
+        float chance = Options.GetRoleChance(role.ThisRoleBase);
+
+        Options.SetRoleChance(role.ThisRoleBase, chance + 1);
+        OnValueChanged(option, role);
+    }
+
+    private static void DecreaseChance(OptionBehaviour option, RoleBase role)
+    {
+        float chance = Options.GetRoleChance(role.ThisRoleBase);
+
+        Options.SetRoleChance(role.ThisRoleBase, chance + 1);
+        OnValueChanged(option, role);
+    }
+
+    private static void OnValueChanged(OptionBehaviour optionnonconv, RoleBase role)
+    {
+        RoleOptionSetting option = optionnonconv as RoleOptionSetting;
+        if (option == null) return;
+
+        option.roleMaxCount = 99;
+        option.roleChance = (int)role.ThisRoleBase.GetChance();
```

Comment:
bruh, "ThisRoleBase" is the base of the role, not the actual role. e.g. Impostor, SS, Scientist etc..

Reply:
Yeah

Reply:
DAYUMMMMMMMMMM